rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```diff
@@ -39,9 +39,11 @@ applications are often leveraged by adversaries to execute code and evade detect
 from = "now-9m"
 index = [
     "logs-endpoint.events.process-*",
+    "endgame-*",
     "logs-endpoint.events.network-*",
-    "winlogbeat-*",
+    "logs-sentinel_one_cloud_funnel.*",
```

Comment:
```suggestion
```
And SentinelOne doesn't populate host.name that is used in the startswith condition